compiler.toml

```
@@ -1,27 +1,16 @@
-name = "Rust Compiler Team"
-description = "Meetings for the compiler team and its working groups"
+name = "Rust Compiler Team (All Events)"
```

```suggestion
name = "Rust Compiler Team + Working Groups"
```

and leave `description` as-is